content/miq_dialogs/miq_provision_kubevirt_dialogs_template.yaml

```diff
@@ -297,9 +297,19 @@
           :default: "1024"
           :data_type: :string
       :display: :show
+    :customize:
+      :description: Customize
+      :fields:
+        :root_password:
+          :description: Cloud User Password
+          :required: true
+          :display: :edit
+          :data_type: :string
+      :display: :show
```

Comment:
@nasark just double-checking, does this show up on the UI now?

Reply:
Yes

<img width="923" alt="Screenshot 2024-09-24 at 2 39 29 PM" src="https://github.com/user-attachments/assets/ce13be1a-9a7f-41fe-8e54-8d70ec57579c">